actors/builtin/miner/miner_test.go

```diff
@@ -3250,40 +3250,6 @@ func TestExtendSectorExpiration(t *testing.T) {
 		})
 		actor.checkState(rt)
 	})
-
-	t.Run("prevents extending old seal proof types", func(t *testing.T) {
```

Comment:
We should keep this test, check that the old behavior applies to network version 10 and then check that v1 seal proof types *can* be extended up to max of 540 days in version 11.

We should also check that v1 seal proof types cannot be extended beyond 540 days.